drivers/start_date_dq/app/views/start_date_dq/warehouse_reports/reports/index.haml

```diff
@@ -0,0 +1,38 @@
+- content_for :title, @report.title
+%h1= @report.title
+= render '/warehouse_reports/breadcrumbs', tab: 'data-quality'
+%p View differences between the client's self-reported date homeless (DateToStreetESSH) and the enrollment entry date.
+= render 'filter'
+- if !@pagy.present?
+  %p.text-center Apply filters to see results.
+- elsif @pagy.count.positive?
+  .d-flex
+    .ml-auto.mb-4
+      = link_to({format: :xlsx}.merge(link_params),{class: 'btn btn-secondary ml-auto'}) do
+        %i.icon-download2
+        Download
+  .d-flex
+    %p!= pagy_info(@pagy, item_name: _('enrollment').pluralize(@pagy.count))
+    %p.ml-auto!= pagy_bootstrap_nav(@pagy)
+  .card
+    %table.mb-0.table.table-striped
+      %thead
+        %tr
+          %th Client
+          - @report.column_names.each do |s|
+            %th= s
+      %tbody
+        - @enrollments.each do |row|
+          - client = row.client
+          - values = @report.column_values(row)
```

Comment:
Days difference needs `number_with_delimiter()` to add the commas.

Reply:
Done